app/src/main/java/com/readystatesoftware/mapviewballoons/BalloonItemizedOverlay.java

```diff
@@ -1,259 +0,0 @@
-/***
- * Copyright (c) 2010 readyState Software Ltd
- * 
- * Licensed under the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License. You may obtain
- * a copy of the License at
- * http://www.apache.org/licenses/LICENSE-2.0
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * 
- */
-
-package com.readystatesoftware.mapviewballoons;
-
```

Comment:
Wouldn't it be better to comment out the code so that we can have it in reference?